tests/network-policy-checks-bad-path/peer_no_specified_fields.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: peer-namespaceselector-test-none
+  namespace: kube-system
+spec:
+  podSelector:
+    matchLabels:
+      app: helm
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            tier: frontend
+  egress:
+    - to: # no specified fields
+        # none value for selector and no other fields
+        - namespaceSelector:
```

Comment:
are you sure this is an error? a `null` labelSelector should select no objects

Reply:
followed the k8s live cluster: 
The NetworkPolicy "peer-namespaceselector-test-none" is invalid: spec.egress[0].to[0]: Required value: must specify a peer

I think because it selects no objects then no peers are specified to "to"

Reply:
It's a bit more complex I think.
If you replace the egress with the following, k8s will accept it.
```
  egress:
    - to: # no specified fields
        # none value for selector and no other fields
        - namespaceSelector:
          podSelector:
            matchLabels:
              tier: frontend
```